distribution/docs/src/main/resources/_contents/_core-concepts/metadata-intro-contents.adoc

```diff
@@ -0,0 +1,59 @@
+
+== Understanding Metadata and Metacards
+
+In ${branding}, <<_glossary_resources,resources>> are the data products, files, reports, or documents of interest to users of the system.
+
+Metadata is information about those resources, organized into a schema to make search possible.
+The ${ddf-catalog} stores this metadata and allows access to it.
+Metacards are single instances of metadata, representing a single resource, in the Catalog.
+Metacards follow one of several schemas to ensure reliable, accurate, and complete metadata.
+Essentially, Metacards function as containers of metadata.
+
+=== Populating Metacards During Ingest
+
+Upon ingest, a transformer will read the metadata from the ingested file and populate the fields of the metacard.
+Exactly how this is accomplished depends on the origin of the data, but most fields (except id) are imported directly.
+
+=== Searching Metadata
+
+${branding} provides the capability to search the Catalog for metadata.
+There are a number of different types of searches that can be performed on the Catalog, and these searches are accessed using one of several interfaces.
+This section provides a very high level overview of introductory concepts of searching with ${branding}.
+These concepts are expanded upon in later sections.
+
+==== Search Types
+
+There are four basic types of metadata search.
+Additionally, any of the types can be combined to create a compound search.
+
+===== Contextual Search
+
+A contextual search is used when searching for textual information.
+It searches all textual metadata fields by default, although it is possible to refine searches to a contextual search on a single attribute.
+It is similar to a Google search over the metadata contained in the Catalog.
+Contextual searches may use wildcards, logical operators, and approximate matches.
+
+===== Spatial Search
+
+A spatial search is used for Area of Interest (AOI) searches.
+Polygon and point radius searches are supported.
+Specifically, the spatial search looks at the metacards' location attribute and coordinates are specified in *WGS 84* decimal degrees.
```

Comment:
This might be endpoint specific, so not sure if we want to list more options here